src/Pages/Answerpage/Answerpage.jsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react'
+import "../Homepage/Homepage.css"
+import SubmitBox from '../../Components/SubmitBox';
+
+export default function Homepage() {
```

Comment:
Answerpage.jsx looks identical to Homepage.jsx
What about using just one of them?